MODULE.bazel

```diff
@@ -48,6 +56,54 @@ node.toolchain(
 )
 use_repo(node, "nodejs", "nodejs_toolchains")
 
+kotlin_extractor_deps = use_extension("//java/kotlin-extractor:deps.bzl", "kotlin_extractor_deps")
+
+# following list can be kept in sync by running `bazel mod tidy` in `codeql`
+use_repo(
+    kotlin_extractor_deps,
+    "codeql_kotlin_defaults",
+    "codeql_kotlin_embeddable",
+    "kotlin-compiler-1.5.0",
```

Comment:
I wonder if we really need this list here - is https://bazel.build/external/extension#repository_names_and_visibility not enough, maybe with an application of `Label` in the correct file?

If not, can you provide a repository dependency chain to illustrate how the build is set up, and where we need these repos?

Reply:
As far as I can see, we generate all kotlin builds in a module extension, so I don't see why we'd need there dependencies declared in the root module.

Reply:
I'll test it, but I _think_ this is required (but made quite easy to maintain using `bazel mod tidy`). I'll test it out though.

Reply:
Yep, this list is required:
```
ERROR: Skipping '//java/kotlin-extractor': error loading package 'java/kotlin-extractor': Unable to find package for @@[unknown repo 'codeql_kotlin_defaults' requested from @@]//:defaults.bzl: The repository '@@[unknown repo 'codeql_kotlin_defaults' requested from @@]' could not be resolved: No repository visible as '@codeql_kotlin_defaults' from main repository.
```

Reply:
> If not, can you provide a repository dependency chain to illustrate how the build is set up, and where we need these repos?

What do you mean by this?